frontend/src/i18n/locales/en.yaml

```diff
@@ -349,6 +349,8 @@ manage:
       Please try again later.
     locked-to-series: >
       The access policy of this video is determined by its series and can't be edited.
+    locked-to-linked-series: >
+      The access policy of this video is determined by its <1>series</1> and can't be edited.
```

Comment:
Why not remove the old translation?

Reply:
It's still used on the upload page. I don't think we need to link the series there? On the other hand it wouldn't hurt either I suppose.